Fix validation uint int sizes (#6434)
ethereum/web3.js: packages/web3-validator/CHANGELOG.md

## Fixed

-   Multi-dimensional arrays are now handled properly when parsing ABIs (#6434)
-   Validator will now properly handle all valid numeric type sizes: intN / uintN where 8 <= N <= 256 and N % 8 == 0 (#6434)

> -   Will now throw SchemaFormatError when unsupported format is given in `convertToZod` method (#6434)

Reviewer: ```suggestion
-   Will now throw SchemaFormatError when unsupported format is passed to `convertToZod` method (#6434)
```
